Cubical/Algebra/Group/EilenbergMacLane/Base.agda

```diff
@@ -91,3 +90,52 @@ elim : {G : AbGroup ℓ} (n : ℕ) {A : EM G n → Type ℓ'}
 elim zero hlev hyp x = hyp x
 elim (suc zero) hlev hyp x = hyp x
 elim (suc (suc n)) hlev hyp = trElim (λ _ → hlev _) hyp
+
+elim2 : ∀ {ℓ''}{G : AbGroup ℓ} {H : AbGroup ℓ'} (n : ℕ) {A : EM G n → EM H n → Type ℓ''}
+        → ((x : _) (y : _) → isOfHLevel (2 + n) (A x y))
+        → ((x : EM-raw G n) (y : EM-raw H n) → A (EM-raw→EM G n x) (EM-raw→EM H n y))
+        → (x : _) (y : _) → A x y
+elim2 n hlev ind =
+  elim n (λ _ → isOfHLevelΠ (2 + n) (λ _ → hlev _ _))
+    λ x → elim n (λ _ → hlev _ _) (ind x)
+
+EM-rawer : ∀ {ℓ} (G : AbGroup ℓ) (n : ℕ) → Type ℓ
```

Comment:
This name is not very informative. Also why not put level at the top of the file?

Reply:
It became EM-raw'... I guess at least it shows its just a version of EM-raw. Can't come up with anything better...